easybuild/easyconfigs/l/lmodule/lmodule-0.4.0-GCCcore-13.3.0-Python-3.12.3.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'PythonPackage'
+
+name = 'lmodule'
+version = '0.4.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/HPC-buildtest/lmodule'
+description = """lmodule is a Python 3 API for Lmod module system. The API can be used end-users and 
+system-administrators. End-users can utilize the API for interacting with module system to run their application. 
+System Administrators can utilize the API for testing their Software Stack and retrieve output from the Lmod spider 
+command."""
+
+toolchain = {'name': 'GCCcore', 'version': '13.3.0'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_PY3_WHL]
+checksums = ['ba1e3317a25fcc0be9676a7a1b27c3fb6b20757d013de0d6fe7965cc23eb0d16']
+
+exts_defaultclass = 'PythonPackage'
+use_pip = True
+
```

Comment:
```suggestion
```

There are no extensions inside this easyconfig, it's not a (python)bundle, and even if it was, this would have been the default anyway